core-bundle/src/EventListener/DataContainer/ContentCompositionListener.php

```diff
@@ -90,14 +89,14 @@ public function generateArticleForPage(DataContainer $dc): void
         $request = $this->requestStack->getCurrentRequest();
         $user = $this->security->getUser();
 
-        // Return if there is no active record (override all)
-        if (!$dc->activeRecord || null === $request || !$user instanceof BackendUser || !$request->hasSession()) {
+        // Return if there is no current record (override all)
+        if (null === ($currentRecord = $dc->getCurrentRecord()) || null === $request || !$user instanceof BackendUser || !$request->hasSession()) {
```

Comment:
For readability that/where the current record is fetched, because it becomes relevant several times below
```suggestion
        $currentRecord = $dc->getCurrentRecord();

        // Return if there is no current record (override all)
        if (null === $currentRecord || null === $request || !$user instanceof BackendUser || !$request->hasSession()) {
```